cmd/kaidump/kaidump.go

```diff
@@ -1,75 +1,93 @@
-package main
```

Comment:
There must be some issues with your Git client because I keep seeing the whole file changed, not the lines you've changed in this commit